.env.example

```diff
@@ -5,10 +5,22 @@ DATABASE_URL=postgres://user:password@localhost:5432/boostagram-viewer
 
 PRISMA_FIELD_ENCRYPTION_KEY=
 
+# default email config will use Mailhog
+EMAIL_SERVER_USER=username
+EMAIL_SERVER_PASSWORD=password
+EMAIL_SERVER_HOST=localhost
+EMAIL_SERVER_PORT=1025
+EMAIL_FROM=noreply@example.com
+
+ORIGIN=http://localhost:8081
+
```

Comment:
I'd ABC-sort the whole file or at least the subsections.

Reply:
fixed, thanks